modules/apps/frontend-js/frontend-js-components-sample-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -0,0 +1,40 @@
+<%--
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<div class="frontend-js-components-sample-web">
+	<liferay-ui:tabs
+		names="Translation Manager"
+		refresh="<%= false %>"
+	>
+
+		<%
+		String[] sections = {"translation_manager"};
+
+		for (int i = 0; i < sections.length; i++) {
+		%>
+
+			<liferay-ui:section>
+				<liferay-util:include page='<%= "/partials/" + sections[i] + ".jsp" %>' servletContext="<%= application %>" />
+			</liferay-ui:section>
```

Comment:
Are we adding more tabs any time soon? Otherwise I'd just simple use <liferay-ui:section> without the loop

Reply:
We also have frontend-js-a11y-sample-web that we could migrate into this generic sample portlet. So I'd like to have it available for this as well as more sub-module test artifacts down the road.

Reply:
I'd prefer to keep it like this to have prepared for what will come.